drivers/motor_driver/motor_driver.c

```diff
@@ -0,0 +1,209 @@
+#include <errno.h>
+
+/* RIOT includes */
+#include <assert.h>
+#include <board.h>
+#include <log.h>
+#include <motor_driver.h>
+
+#define ENABLE_DEBUG    (0)
+#include <debug.h>
+
+#ifdef MOTOR_DRIVER_NUMOF
+
+#define MOTOR_DRIVER_MODE        PWM_LEFT
```

Comment:
Because I did not see some cases where PWM alignment is important for motor driving. So I fixed it to PWM_LEFT.
Do you some case where it could be useful to change it ?

Reply:
Well, it depends on what other components there are on the system. As I said before, I see this module as a generic H-bridge driver-driver (it supports all of the functionality of common H-bridge drivers, right?)

For example, by periodically changing the period one could modulate a sine wave. In this case a centered PWM is desirable as it makes for a symettric signal with reduced harmonic content.